--- libcanberra-0.30/src/canberra-system-bootup.service.in.0001~	2011-02-23 02:51:00.000000000 +0100
+++ libcanberra-0.30/src/canberra-system-bootup.service.in	2013-03-31 16:34:49.228134856 +0200
@@ -7,7 +7,7 @@ Conflicts=shutdown.target
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=/usr/share/sounds/freedesktop/stereo/system-bootup.oga
+ConditionPathExists=/usr/share/sounds/moondrake/stereo/system-bootup.oga
 
 [Service]
 Type=oneshot
--- libcanberra-0.30/src/canberra-system-shutdown-reboot.service.in.0001~	2012-05-15 12:17:34.000000000 +0200
+++ libcanberra-0.30/src/canberra-system-shutdown-reboot.service.in	2013-03-31 16:34:49.228134856 +0200
@@ -5,8 +5,8 @@ Before=shutdown.target
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=|/usr/share/sounds/freedesktop/stereo/system-shutdown.oga
-ConditionPathExists=|/usr/share/sounds/freedesktop/stereo/system-shutdown-reboot.oga
+ConditionPathExists=|/usr/share/sounds/moondrake/stereo/system-shutdown.oga
+ConditionPathExists=|/usr/share/sounds/moondrake/stereo/system-shutdown-reboot.oga
 
 [Service]
 Type=oneshot
--- libcanberra-0.30/src/canberra-system-shutdown.service.in.0001~	2011-02-23 02:51:00.000000000 +0100
+++ libcanberra-0.30/src/canberra-system-shutdown.service.in	2013-03-31 16:34:49.228134856 +0200
@@ -5,7 +5,7 @@ Before=shutdown.target
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=/usr/share/sounds/freedesktop/stereo/system-shutdown.oga
+ConditionPathExists=/usr/share/sounds/moondrake/stereo/system-shutdown.oga
 
 [Service]
 Type=oneshot
--- libcanberra-0.30/src/sound-theme-spec.c.0001~	2013-03-31 16:48:11.243105196 +0200
+++ libcanberra-0.30/src/sound-theme-spec.c	2013-03-31 16:48:41.739444364 +0200
@@ -34,7 +34,7 @@
 #include "llist.h"
 #include "cache.h"
 
-#define DEFAULT_THEME "freedesktop"
+#define DEFAULT_THEME "moondrake"
 #define FALLBACK_THEME "freedesktop"
 #define DEFAULT_OUTPUT_PROFILE "stereo"
 #define N_THEME_DIR_MAX 8
