diff -Naur libcanberra-0.29/src/canberra-system-bootup.service.in libcanberra-0.29.tpg/src/canberra-system-bootup.service.in
--- libcanberra-0.29/src/canberra-system-bootup.service.in	2011-02-23 01:51:00.000000000 +0000
+++ libcanberra-0.29.tpg/src/canberra-system-bootup.service.in	2013-02-02 10:42:28.477013045 +0000
@@ -7,7 +7,7 @@
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=/usr/share/sounds/freedesktop/stereo/system-bootup.oga
+ConditionPathExists=/usr/share/sounds/moondrake/stereo/system-bootup.oga
 
 [Service]
 Type=oneshot
diff -Naur libcanberra-0.29/src/canberra-system-shutdown-reboot.service.in libcanberra-0.29.tpg/src/canberra-system-shutdown-reboot.service.in
--- libcanberra-0.29/src/canberra-system-shutdown-reboot.service.in	2012-05-15 10:17:34.000000000 +0000
+++ libcanberra-0.29.tpg/src/canberra-system-shutdown-reboot.service.in	2013-02-02 10:42:48.341543848 +0000
@@ -5,8 +5,8 @@
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=|/usr/share/sounds/freedesktop/stereo/system-shutdown.oga
-ConditionPathExists=|/usr/share/sounds/freedesktop/stereo/system-shutdown-reboot.oga
+ConditionPathExists=|/usr/share/sounds/moondrake/stereo/system-shutdown.oga
+ConditionPathExists=|/usr/share/sounds/moondrake/stereo/system-shutdown-reboot.oga
 
 [Service]
 Type=oneshot
diff -Naur libcanberra-0.29/src/canberra-system-shutdown.service.in libcanberra-0.29.tpg/src/canberra-system-shutdown.service.in
--- libcanberra-0.29/src/canberra-system-shutdown.service.in	2011-02-23 01:51:00.000000000 +0000
+++ libcanberra-0.29.tpg/src/canberra-system-shutdown.service.in	2013-02-02 10:42:59.652846134 +0000
@@ -5,7 +5,7 @@
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=/usr/share/sounds/freedesktop/stereo/system-shutdown.oga
+ConditionPathExists=/usr/share/sounds/moondrake/stereo/system-shutdown.oga
 
 [Service]
 Type=oneshot
