diff -Naur libcanberra-0.30/src/canberra-system-bootup.service.in libcanberra-0.30.tpg/src/canberra-system-bootup.service.in
--- libcanberra-0.30/src/canberra-system-bootup.service.in	2011-02-23 01:51:00.000000000 +0000
+++ libcanberra-0.30.tpg/src/canberra-system-bootup.service.in	2013-08-20 10:09:57.600804064 +0000
@@ -7,11 +7,11 @@
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=/usr/share/sounds/freedesktop/stereo/system-bootup.oga
+ConditionPathExists=/usr/share/sounds/ia_ora/stereo/ia_ora-startup.oga
 
 [Service]
 Type=oneshot
-ExecStart=-@bindir@/canberra-boot system-bootup
+ExecStart=-@bindir@/canberra-boot ia_ora-startup
 
 [Install]
 WantedBy=sound.target
diff -Naur libcanberra-0.30/src/canberra-system-shutdown-reboot.service.in libcanberra-0.30.tpg/src/canberra-system-shutdown-reboot.service.in
--- libcanberra-0.30/src/canberra-system-shutdown-reboot.service.in	2012-05-15 10:17:34.000000000 +0000
+++ libcanberra-0.30.tpg/src/canberra-system-shutdown-reboot.service.in	2013-08-20 09:20:03.326898429 +0000
@@ -5,12 +5,11 @@
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=|/usr/share/sounds/freedesktop/stereo/system-shutdown.oga
-ConditionPathExists=|/usr/share/sounds/freedesktop/stereo/system-shutdown-reboot.oga
+ConditionPathExists=|/usr/share/sounds/ia_ora/stereo/ia_ora-shutdown.oga
 
 [Service]
 Type=oneshot
-ExecStart=-@bindir@/canberra-boot system-shutdown-reboot
+ExecStart=-@bindir@/canberra-boot ia_ora-shutdown
 
 [Install]
 WantedBy=reboot.target kexec.target
diff -Naur libcanberra-0.30/src/canberra-system-shutdown.service.in libcanberra-0.30.tpg/src/canberra-system-shutdown.service.in
--- libcanberra-0.30/src/canberra-system-shutdown.service.in	2011-02-23 01:51:00.000000000 +0000
+++ libcanberra-0.30.tpg/src/canberra-system-shutdown.service.in	2013-08-20 09:20:38.589835390 +0000
@@ -5,11 +5,11 @@
 
 # A little optimization under the assumption that we are using the
 # standard freedesktop theme
-ConditionPathExists=/usr/share/sounds/freedesktop/stereo/system-shutdown.oga
+ConditionPathExists=/usr/share/sounds/ia_ora/stereo/ia_ora-shutdown.oga
 
 [Service]
 Type=oneshot
-ExecStart=-@bindir@/canberra-boot system-shutdown
+ExecStart=-@bindir@/canberra-boot ia_ora-shutdown
 
 [Install]
 WantedBy=halt.target poweroff.target
diff -Naur libcanberra-0.30/src/sound-theme-spec.c libcanberra-0.30.tpg/src/sound-theme-spec.c
--- libcanberra-0.30/src/sound-theme-spec.c	2010-02-19 23:39:40.000000000 +0000
+++ libcanberra-0.30.tpg/src/sound-theme-spec.c	2013-08-20 09:23:01.578634957 +0000
@@ -34,7 +34,7 @@
 #include "llist.h"
 #include "cache.h"
 
-#define DEFAULT_THEME "freedesktop"
+#define DEFAULT_THEME "ia_ora"
 #define FALLBACK_THEME "freedesktop"
 #define DEFAULT_OUTPUT_PROFILE "stereo"
 #define N_THEME_DIR_MAX 8
